### ISSUE TITLE
Soul scythes cannot hear music from the dance machine?

### ISSUE THREAD

**Issue by plutori** [tgstation/tgstation#76385]
Reporting client version: 514.1589

## Round ID:
209238

## Reproduction:

1. Accept the offer as a dead person to become the spirit of someone's soul scythe
2. Enter the Uncle Pete's Rollerdome, or, presumably, Disco Inferno (The reporting shift was Rollerdome)
3. Have someone who actually has hands play music on the radiant dance machine mark V
4. Be very sad when you cannot hear the music.

**Comment by TheVekter** [comment]
Pete's and Disco Inferno use the same disco ball, so it'll happen on both.

Additionally, the disco ball is just a resprited jukebox that forces people to dance, so this probably also happens with the jukebox.

**Comment by Sealed101** [comment]
could either be the machine not picking up the scythe's soul mob or it could be failing the `get_dist()` check, since the mob is in an item and could be on a person
#### relevant code
https://github.com/tgstation/tgstation/blob/e6f545c3e30075c4e6acfe6425f5171fad3e38cf/code/game/machinery/dance_machine.dm#L452-L476

**Comment by MrMelbert** [comment]
> ould either be the machine not picking up the scythe's soul mob or it could be failing the get_dist() check, since the mob is in an item and could be on a person

Correct, the "mob" is in an item's contents and thus is not included in `range()`
